src/main/scala/tile/BaseTile.scala

```diff
@@ -103,10 +103,9 @@ trait HasNonDiplomaticTileParameters {
     val f = if (tileParams.core.fpu.nonEmpty) "f" else ""
     val d = if (tileParams.core.fpu.nonEmpty && tileParams.core.fpu.get.fLen > 32) "d" else ""
     val c = if (tileParams.core.useCompressed) "c" else ""
-    val b = if (tileParams.core.useBitManip) "b" else ""
     val v = if (tileParams.core.useVector) "v" else ""
     val x = tileParams.core.customIsaExt.map(s => s"_$s").getOrElse("")
-    s"rv${p(XLen)}$ie$m$a$f$d$c$b$v$x"
+    s"rv${p(XLen)}$ie$m$a$f$d$c$v$x"
```

Comment:
Shouldn't `bitmanip` be reflected somehow in the dts? I don't know what the exact correct isa string composiiton should be, but it should be implemented here.

Reply:
Added.